app/api.py

```diff
@@ -791,3 +798,21 @@ def get_element_text(self, doc, tag_name):
             return text.encode('utf-8')
         else:
             return None
+
+    #Send hit to GA
+    def gamp(e,ec):
```

Comment:
Use more descriptive name for the method name and parameter names (gamp, e, ec).